plugins/activity-resources/src/components/TxView.svelte

```diff
@@ -163,13 +175,14 @@
     }
     return false
   }
+
   let hasMessageType = false
   $: updateMessageType(model, tx).then((res) => {
     hasMessageType = res
   })
   $: isComment = viewlet && viewlet?.editable
   $: isAttached = isAttachment(tx.tx)
-  $: isMentioned = isMention(tx.tx.objectClass)
+  $: isMentioned = isMention()
```

Comment:
We still need this to be passed to extension somehow, overwize it could stuck and will not be updated.

Reply:
I think we need to pass extensions here

Reply:
as parameter

Reply:
Added.